app/views/spree/admin/authentication_methods/index.html.erb

```diff
@@ -25,7 +25,9 @@
       <col style="width: 20%" />
     </colgroup>
     <thead data-hook="admin_social_methods_index_headers">
+      <th><%= I18n.t('spree.store') %></th>
```

Comment:
Not so relevant, but can't find why this changes are not shown in the app.

Reply:
Its showing on my machine. Only `I18n.t('spree.social_site')` key is missing. I am adding it to this PR

Reply:
@junan this is in the index not in the edit.